src/CurveController.sol

```diff
@@ -0,0 +1,243 @@
+/// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
rename it to `LongPositionHandler` curve controller doesn't match with our nomenclature